src/types.py

```diff
@@ -21,6 +20,29 @@ def __init__(self, **kwargs):
         for key, value in kwargs.items():
             setattr(self, key, value)
 
+class ScoringSummaryType(ObjectType):
+    """
+    Represents a single scoring summary entry in a game.
```

Comment:
Could you add documentation about each attribute as well? You can reference my code for examples.